CMakeLists.txt

```diff
@@ -195,6 +195,28 @@ include_directories(${VCL_INCLUDE_DIRS} ${VXLCORE_INCLUDE_DIRS})
 include(${CMAKE_CURRENT_LIST_DIR}/config/cmake/config/VXLIntrospectionConfig.cmake)
 
 #-------------------------------------------------------------------
+# This block should come after VXLIntrospectionConfig.cmake has been executed.
+if(VCL_HAS_LFS OR WIN32)
+  option( VXL_USE_LFS "Should VXL use Large File Support?" NO)
+  mark_as_advanced( VXL_USE_LFS )
+endif()
+
+if(VXL_USE_LFS)
+  if(WIN32)
+    # TODO: MS Version Support
+    #  message( SEND_ERROR "Sorry - Large File Support is not quite working on Win32 yet. Turning VXL_USE_LFS off")
```

Comment:
It could be done as a separate commit of this PR or as a separate PR.  I'm fine with either.